src/vs/editor/contrib/suggest/suggestController.ts

```diff
@@ -174,12 +178,30 @@ export class SuggestController implements IEditorContribution {
 			}
 		}));
 		this._toDispose.add(this._editor.onDidBlurEditorWidget(() => {
-			if (!_sticky) {
+			if (!_sticky && !this._widget.getValue().isDetailsFocused) {
 				this._model.cancel();
 				this._model.clear();
 			}
 		}));
 
+		this._toDispose.add(this._widget.getValue().onDetailsKeyDown(e => {
```

Comment:
Should we maybe simply forward the event to the regular keydown handler instead of duplicating the handling here?

Reply:
@joaomoreno I can't do that, as those events only trigger proper action when editor is being focused. For this to work, details element has to be in focus when selection happens.

Reply:
I take that back. I should find a way to do that, otherwise many keyboards shortcuts won't work while details is in focus.